UniversalAlgebra/Overture/Preliminaries.lagda

```diff
@@ -141,6 +141,51 @@ A × B = Σ[ x ꞉ A ] B
 ```
 
 
+The following types will also come in handy on occasion.
+
+\begin{code}
+
+record Σω {𝓤} (A : Type 𝓤) (B : A → Setω) : Setω where
```

Comment:
That this might be useful is very scary!

Reply:
Agreed.  I have not used this.  I was just trying it out to see what was possible.  The comments are out-of-date.  Sorry if they are misleading.